.eslintrc.js

```diff
@@ -1,3 +1,16 @@
 module.exports = {
-  "extends": "google"
-};
\ No newline at end of file
+  'extends': 'google',
+  'parser': 'babel-eslint',
```

Comment:
do not replace the original parser, try to use the default one and solve the es6 and jsx features with it

https://eslint.org/docs/user-guide/configuring#specifying-parser-options